### ISSUE TITLE
Dataset Ideas to Expand Ability to meet Equity Goals

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#702]
![Screen Shot 2021-12-10 at 12 24 48 PM](https://user-images.githubusercontent.com/66691020/145873866-afead05a-c56a-419b-b8af-3e170c976e26.png)

Current DataSets that could maybe be used in a more targeted lesson:

- Gerrymandering
- Marijuana Laws & Arrests
- Refugees
- College Majors
- US Jobs

We are about to have a group of undergrads support us in expanding our datasets. They will start with the same prompt, but if they don't come up with an idea of their own, we will need to assign some.  

Brainstorming here about relevant issues that there may be good data on.
- Brown University connections to slavery [https://slaveryandjustice.brown.edu/report](https://slaveryandjustice.brown.edu/report)
- Brown University response to diversifying society - we could inquire with them re: datasets [https://bulletin.brown.edu/centerforthestudyofraceandethnicityinamerica/](https://bulletin.brown.edu/centerforthestudyofraceandethnicityinamerica/)
- Immigrants in the US work force
- Colleges - tuition - diversity of applicants / accepted students percentages - community/private/state - small/medium/large - football budget - number of students at a college columns for year and college name - row: Brown University 1960
- average class sizes / zipcode / median and mean income of zipcode / money spent per student / student-teacher ratio / access to music / art / shop / CS education in schools around the country 
- wealth gap - annual earnings / equity / hours worked... 

> Perhaps no statistic better illustrates the enduring legacy of our country’s shameful history of treating black people as sub-citizens, sub-Americans, and sub-humans than the wealth gap.

- predatory lending in Baltimore 
> In 2005, Wells Fargo promoted a series of Wealth Building Strategies seminars. Dubbing itself “the nation’s leading originator of home loans to ethnic minority customers,” the bank enrolled black public figures in an ostensible effort to educate blacks on building “generational wealth.” But the “wealth building” seminars were a front for wealth theft. In 2010, the Justice Department filed a discrimination suit against Wells Fargo alleging that the bank had shunted blacks into predatory loans regardless of their creditworthiness. This was not magic or coincidence or misfortune. It was racism reifying itself. According to The New York Times, affidavits found loan officers referring to their black customers as “mud people” and to their subprime products as “ghetto loans.”
>   
>   “We just went right after them,” Beth Jacobson, a former Wells Fargo loan officer, told The Times. “Wells Fargo mortgage had an emerging-markets unit that specifically targeted black churches because it figured church leaders had a lot of influence and could convince congregants to take out subprime loans.”
>   
>   In 2011, Bank of America agreed to pay $355 million to settle charges of discrimination against its Countrywide unit. The following year, Wells Fargo settled its discrimination suit for more than $175 million. But the damage had been done. In 2009, half the properties in Baltimore whose owners had been granted loans by Wells Fargo between 2005 and 2008 were vacant; 71 percent of these properties were in predominantly black neighborhoods.

- redlining [contract buyers league chicago 9 minute clip](https://youtu.be/YxPX_uJ36bg) Redlining was not officially outlawed until 1968, by the Fair Housing Act. By then the damage was done—and reports of redlining by banks have continued.
- Dreamers
- Dollar street from Gapminder.org [wage calculations](https://drive.google.com/drive/folders/0B9jWD65HiLUnRm5ZNWlMSU5GNEU?resourcekey=0-4rjWstzby3z96urmt8QgpA)
- professional sports earnings injuries / by age / sex / race / geography / education / coach v player v ref v sportscaster etc.
- Environmental justice - NYC data asthma rates / bus depot location / vegetable access
- Haitian Revolution
- military veterans
- aids funding pre class action lawsuit?
- Reparations 
> Scholars have long discussed methods by which America might make reparations to those on whose labor and exclusion the country was built. In the 1970s, the Yale Law professor Boris Bittker argued in The Case for Black Reparations that a rough price tag for reparations could be determined by multiplying the number of African Americans in the population by the difference in white and black per capita income. That number—$34 billion in 1973, when Bittker wrote his book—could be added to a reparations program each year for a decade or two. Today Charles Ogletree, the Harvard Law School professor, argues for something broader: a program of job training and public works that takes racial justice as its mission but includes the poor of all races.

**Comment by flannery-denny** [comment]
_Three categories of data dominated the responses. Topics related
to food and health came first at 20.8%. Two were about New York
City restaurants, but all the rest had a health dimension: nutrition
in fast foods, sodas, (breakfast) cereals, snacking, and cancer rates.
Equally popular was a cluster on crime and policing, mostly about
stop-and-frisk and marijuana arrest rates (comparing Black versus
white, rural versus urban, etc.): in short, politically important and
sensitive topics (some of which may have been especially salient in
light of 2020’s Black Lives Matter protests). Close behind was the
category of sports and entertainment (16.4%), but spread between
baseball, basketball, track-and-field, and video game reviews. A
dataset of the “Top 100 Movies” attracted 7% of students, followed
by a long tail: data about specific states, the environment, animals,
college majors, and charter schools each had more than one user.
In short, we see a broad spread of topics that could reasonably
be described as personally meaningful._

**Comment by retabak** [comment]
Maybe something on college admissions pre / post required standardized testing? Here's an [article](https://www.forbes.com/sites/kimelsesser/2020/04/27/sat-act-policies-may-improve-diversity-at-colleges-and-universities/?sh=28fcbe7d3bd5) and a [study](https://www.nacacnet.org/globalassets/documents/publications/research/defining-access-report-2018.pdf) and also a pretty interesting [This American Life episode](https://www.thisamericanlife.org/734/the-campus-tour-has-been-cancelled)

I'll likely think of more!

**Comment by flannery-denny** [comment]
Rec from Shriram: [https://www.datacommons.org/](https://www.datacommons.org/)

**Comment by flannery-denny** [comment]
@schanzer @retabak I have 15 undergrads signed up to work on datasets over winter break! and 14 of them want to be assigned a focus. All input welcome! Trying to get back to them tomorrow.

Particularly useful would be to suggest columns that you would like to see included. And any sources for data on your radar.

Current ideas from their interests and our previous list.
1. **Colleges** 
> _COLUMNS:_ tuition, diversity of applicants, accepted students percentages, community/private/state, size, football budget, year, college name 
> _ROWS:_ Brown University 1960, Brown University 1961, etc.
2. **Immigrants in the Workforce** 
> _COLUMNS:_ tuition, diversity of applicants, accepted students percentages, community/private/state, size, football budget, year, college name 
> _ROWS:_ Brown University 1960, Brown University 1961, etc.
> _POSSIBLE SOURCES:_
3. **Air Pollution:** (suggested by an undergrad)
> _COLUMNS:_ population, median income, %white, %other races, asthma rate,  
> _ROWS:_ zipcode
> _POSSIBLE SOURCES:_
4. **Covid in relation to other factors such as economic impact on different classes or infection rates by economic class** (suggested by an undergrad)
> _COLUMNS:_ 
> _ROWS:_ 
> _POSSIBLE SOURCES:_
5. **College Admissions / Standardized Testing** (suggested by Rachel & an undergrad)
> _COLUMNS:_ 
> _ROWS:_ 
6. **Books**(suggested by an undergrad as a topic we're missing)- I wonder about books that are required reading in school, books that have been banned, books that are ny times best sellers, diversity in children's books... anyone have any ideas about what the best entry point is?
> _COLUMNS:_ 
> _ROWS:_ 
> _POSSIBLE SOURCES:_
7. **Diversity in Hollywood** (Suggested by Rachel)
> _COLUMNS:_ 
> _ROWS:_ 
> _POSSIBLE SOURCES:_
8. **Earthquakes** 
> _COLUMNS:_ 
> _ROWS:_ 
> _POSSIBLE SOURCES:_ Put them in touch with Kelian who is about to earn his phd in geology and has offered to support an undergrad.
9. **Diversity in Hollywood** 
> _COLUMNS:_ 
> _ROWS:_ 
> _POSSIBLE SOURCES:_
10. **Immigrants in the Workforce** 
> _COLUMNS:_ tuition, diversity of applicants, accepted students percentages, community/private/state, size, football budget, year, college name 
> _ROWS:_ Brown University 1960, Brown University 1961, etc.
> _POSSIBLE SOURCES:_
11. **Earthquakes** 
> _COLUMNS:_ 
> _ROWS:_ 
> _POSSIBLE SOURCES:_ Put them in touch with Kelian who is about to earn his phd in geology and has offered to support an undergrad.
12. **Obesity:** (suggested by an undergrad)
> _COLUMNS:_ obesity rate, average commute, , %other races, asthma rate, 
> _ROWS:_ zipcode
> _POSSIBLE SOURCES:_ [county health rankings](https://www.countyhealthrankings.org/)
13. **Diversity in Hollywood** 
> _COLUMNS:_ 
> _ROWS:_ 
> _POSSIBLE SOURCES:_
14. **Diversity in Hollywood** 
> _COLUMNS:_ 
> _ROWS:_ 
> _POSSIBLE SOURCES:_

MAYBE. **TikTok Trends** (suggested by an undergrad - if they can sell me on it I'm in)
> _COLUMNS:_ 
> _ROWS:_ 
> _POSSIBLE SOURCES:_

**Comment by flannery-denny** [comment]
@schanzer Several of the datasets we'll be building will use zipcodes as rows.

There are 41,692 ZIP Codes in the united states. Ideas on how to select a manageable number of them to include. Should this be random? If so, is there a simple way to randomly select rows from a csv file that you know of?

**Comment by schanzer** [comment]
@flannery-denny to randomize rows...

1. Select all rows
2. Right-click and select "randomize range"

Then select the top _n_ rows, which represent _n_ random rows pulled from the population

**Comment by flannery-denny** [comment]
awesome. thanks!

**Comment by flannery-denny** [comment]
@schanzer I want to make an organized list of the ideas that were generated for future datasets and close this issue. Would you prefer the list in a git issue or as a google doc in the datasets folder?
